core-services/prow/02_config/_config.yaml

```diff
@@ -48,6 +48,10 @@ branch-protection:
       repos:
         monstorak-operator:
           protect: false
+    opendatahub-io:
+      repos:
+        odh-manifests:
+          protect: false
```

Comment:
You do not need this until you add Prow jobs, please remove this.

Reply:
Done.  Thanks.